content/guides/typescript.md

```diff
@@ -48,9 +60,13 @@ See [TypeScript's documentation](https://www.typescriptlang.org/docs/handbook/ts
 
 __webpack.config.js__
```

Comment:
Could you make this a diff instead? There's already a config file, so it might be good to show what changed form the "Getting Started" end state of the file.